dhis-2/dhis-api/src/main/java/org/hisp/dhis/security/apikey/ApiToken.java

```diff
@@ -44,86 +48,46 @@
 /**
  * @author Morten Svanæs <msvanaes@dhis2.org>
  */
+@Getter
+@Setter
+@Builder( toBuilder = true )
 @JacksonXmlRootElement( localName = "apiToken", namespace = DxfNamespaces.DXF_2_0 )
-public class ApiToken extends BaseIdentifiableObject implements MetadataObject
+public final class ApiToken extends BaseIdentifiableObject implements MetadataObject
```

Comment:
I am wondering if `final` on a hibernate mapped class could pose an issue for the hibernate proxy.